### ISSUE TITLE
Search: Select menus not associated correctly with the labels

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#53]
URL: https://hcdev.openplus.ca/en/user/203/search
User Agent: Chrome, Firefox
Language: Both

Issue Description: A couple of select menus (drop down menus) are not done correctly and there seems to be a missing label as there are 2 select menus in the code. The label belongs to the 2nd menu for both of the select menus. Issue is with Gagegory and Issue type fields. See screenshot.

Steps to reproduce: Used WAVE toolbar first then inspected code to verify the issue.

Expected results: Each label should have one select menu and "for" attribute in the label tag should be associated/matched with "id" attribute in the select tag.

![image](https://user-images.githubusercontent.com/14280062/128887208-31baa02b-db30-4419-a223-20071f2fabb2.png)

**Comment by echouin** [comment]
@qanitah ﻿this has been reported, it's an issue with the module ﻿https://www.drupal.org/project/shs/issues/3106568. We will mark this as something to revisit once the module has been updated.

**Comment by hydegeoff** [comment]
The original issue is a problem.  All drop-downs require labels.  
To reproduce the issue: 
1.  Login 
2. From the dashboard, select the search tab.
3.  activate the wave plugin 
4. Turn-off CSS will reproduce the exact screenshot above 
Problem: two drop-down are missing labels

**Comment by echouin** [comment]
@hydegeoff this was a Drupal Module issue. Issue was reported to the maintainer. No further work will be able to be done by OP to fix this issue.

**Comment by hydegeoff** [comment]
@echouin, @mmiron-HC  As per your comments above, I have set this issue to won't fix

**Comment by davidgotlieb** [comment]
For consistency this should be tagged as "reported" along with all the other ones where we have reported it to be fixed by Drupal or the owner of the feature.
